packages/taquito-http-utils/src/taquito-http-utils.ts

```diff
@@ -52,9 +52,7 @@ export class HttpRequestFailed extends Error {
   public name = 'HttpRequestFailed';
 
   constructor(
-    public message: string,
-    public url: string,
-    public method: HttpRequestOptions['method']
+    public message: string
```

Comment:
please ignore this change and the integration tests, it's from the most recent master which I made a mistake to branch from. My changes are only under `taquito-rpc/src`